apps/portals/adknowledgeportal/src/config/synapseConfigs/studies.ts

```diff
@@ -46,219 +56,33 @@ export const studyCardConfiguration: CardConfiguration = {
     ],
   },
 }
-const columnAliases = {
+export const studyColumnAliases = {
   DataType_All: 'Data Types',
   Data_Contributor: 'Data Contributor',
   Study_Description: 'Study Description',
   Study_Name: 'Study Name',
   Number_of_Individuals: 'Individuals',
   'Grant Number': 'Grant',
 }
-const studies: SynapseConfig = {
-  name: 'QueryWrapperPlotNav',
-  props: {
-    rgbIndex,
-    columnAliases,
-    sql: studiesSql,
-    name: 'Studies',
-    shouldDeepLink: true,
-    cardConfiguration: studyCardConfiguration,
-    searchConfiguration: {
-      searchable: [
-        'Study_Name',
-        'Study_Description',
-        'DataType_All',
-        'studyFocus',
-        'Data_Contributor',
-        'specimenType',
-        'Species',
-        'Grant Number',
-        'Program',
-      ],
-    },
-  },
-}
 
-export const studiesDetailsPageProps: DetailsPageProps = {
```

Comment:
DetailsPage configs were moved to the new page (in this case StudyDetailsPage.tsx)